### DISCUSSION TITLE
Overlay idea

### DISCUSSION THREAD

**Discussion by MateusMourao** [C1rdec/PoeLurker#384] [Ideas]
guys, when we want to buy something like currency we know the value but we dont know what people will trade with us.
exemple: im buying chromatic x1?? = 20c
i send 15 messages to 15 people, one will send me party but i dont remember how many chromatic that person will trade with my chaos
you can add overlay above os below the Chaos with information how many currency that guy will trade with us?
dont know if u unterstood, sorry about my bad english :P

like this:

HP ball
20c
for
120chro
HP ball

**Comment by C1rdec** [comment]
I'm not sure what's the value of showing this information it will bring more confusion than anything else.

You'll know when you will trade with the seller
